apps/mull-api/src/app/channel/channel.resolver.ts

```diff
@@ -1,20 +1,27 @@
 import { UseGuards } from '@nestjs/common';
 import { Args, Int, Mutation, Query, Resolver } from '@nestjs/graphql';
 import { AuthGuard } from '../auth/auth.guard';
-import { Channel } from '../entities';
+import { Channel, DirectMessageChannel, EventChannel } from '../entities';
+import { UserInput } from '../user/inputs/user.input';
 import { ChannelService } from './channel.service';
-import { CreateChannelInput } from './inputs/channel.input';
+import { CreateDmChannelInput, CreateEventChannelInput } from './inputs/channel.input';
 @Resolver('Channel')
 export class ChannelResolver {
   constructor(private readonly channelService: ChannelService) {}
 
   @Mutation(/* istanbul ignore next */ () => Boolean)
   @UseGuards(AuthGuard)
-  async createChannel(
-    @Args('input') input: CreateChannelInput,
+  async createEventChannel(
+    @Args('input') input: CreateEventChannelInput,
     @Args('eventId', { type: /* istanbul ignore next */ () => Int }) eventId?: number
   ) {
-    return this.channelService.createChannel(input, eventId);
+    return this.channelService.createEventChannel(input, eventId);
+  }
+
+  @Mutation(/* istanbul ignore next */ () => DirectMessageChannel)
```

Comment:
Noticed here we return a Channel but in `createEventChannel` we return a Boolean. Should we make it so both return a channel or both return a boolean to be consistent?

Reply:
Sure, we will return it for all creates/deletes in case we need it in the future